source/_patterns/20-molecules/article-teaser/article-teaser.twig

```diff
@@ -5,7 +5,7 @@
 
     <div class="article-teaser__content">
       <p class="article-teaser__kicker">Schwinger im Joggeli</p>
-      <h3 class="article-teaser__title">Artikel-Titel</h3>
+      <p class="article-teaser__title">Artikel-Titel</p>
```

Comment:
warum kein Titel mehr im Artikel-Teaser?

Reply:
Grund dafür: «Anforderung: Überschrift muss nachfolgenden Inhalt aufweisen
Wenn nach Überschriften auf Seiten kein zugeordneter Inhalt folgt, dann stellt dies keine korrekte semantische Struktur dar. In der Folge ist dieser Aufbau verwirrend für Screenreader-NutzerInnen.» aus Accessibility Review_ZFA_Artikel-Seiten_SRF_v1.0.pdf S. 12

Reply:
A2A & Related Articles werden im CMS mit sichtbaren Titeln eingeleitet und sind verlinkt